repository/w.json

```diff
@@ -370,11 +370,22 @@
 			"previous_names": ["White Night Theme"],
 			"releases": [
 				{
-					"sublime_text": "*",
+					"sublime_text": ">=3149",
 					"branch": "master"
 				}
 			]
 		},
+		{
+			"name": "Whitelines",
+			"details": "https://github.com/TEParsons/Whitelines",
+			"labels": ["colour scheme", "color scheme", "stationery", "light", "orange", "grey", "silver"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": "Whitelines"
```

Comment:
I think you want to set `tags` to `true`. Custom tag names only really make sense when you have two separate releases for different ST version.

What you should do to fix the bot issue is to create a tagged release on github.

Reply:
Thanks, I think this should do it?